src/features/search/components/SearchBar.tsx

```diff
@@ -79,13 +81,14 @@ type SearchBarProps = {
     search: (query: string) => void,
     query: string,
     setQuery: Dispatch<SetStateAction<string>>,
-    stockList: StockListItem[],
     setSelectedStock: Dispatch<SetStateAction<string[]>>,
     selectedStock: string[],
     socket: SocketIOClient.Socket,
 }
 
-export const SearchBar = memo<SearchBarProps>(({isOpen, toggleIsOpen, inputSelect, dropSelect, search, query, setQuery, stockList, setSelectedStock, selectedStock, socket}) => {
+export const SearchBar = memo<SearchBarProps>(({isOpen, toggleIsOpen, inputSelect, dropSelect, search, query, setQuery, setSelectedStock, selectedStock, socket}) => {
```

Comment:
Just noticed that this has `inputRef` and `dropRef`.
I typically find it bad practice to bleed refs through the component tree, if it can be at all helped.
I'd make a GitHub issue to look into whether this can be removed (protip: I believe it can with an intermediate function before calling `setSelectedStock`)

Reply:
raised as an issue and will look into fixing this in future pr!